### ISSUE TITLE
[Request] Built-in Adblocking

### ISSUE THREAD

**Issue by JadenJSJ** [ttalvitie/browservice#45]
It would be super handy if a adblocker was builtin or even better extension support, 
I know that i'm asking to much but I think everyone that has used browservice has at least used 1 extension on their main browser.

This on is the best so far (Its even open-source just like browservice)
https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm?hl=en

**Comment by ttalvitie** [comment]
As I say in [this comment](https://github.com/ttalvitie/browservice/issues/18#issuecomment-784879027), extensions are not currently a priority for me. Adblocking in general is outside my expertise and interests. Nevertheless, it seems that adblocking and extensions are very important for many users. I hope that someone else can take up the task and produce a pull request or a proof-of-concept.

Have you tried network-level ad blockers? They probably are not as good as adblocking browser extensions, but could serve as a stopgap solution.

**Comment by JadenJSJ** [comment]
Yes, I used pi-hole on a docker container, but I couldn't get the dns to point to pihole, it kept pointing to the government's dns.
(They inforced it to block all crypto currency sites and more)
